src/routes/console/project-[project]/settings/migrations/(import)/wizard.svelte

```diff
@@ -103,4 +103,4 @@
     });
 </script>
 
-<Wizard title="Create Project" {steps} on:exit={onExit} on:finish={onFinish} />
+<Wizard title="Create Migration" {steps} on:exit={onExit} on:finish={onFinish} />
```

Comment:
I think the second word should be lower case right? :thinking:

Reply:
@ArmanNik @TGlide this is fine. Because the 2nd word here act as a noun it should start with a capital letter. @gewenyu99 has created PR to fix this all over the console.